### ISSUE TITLE
Add opening screen/title screen

### ISSUE THREAD

**Issue by michaelosbornegit** [Computational-Worlds-Blue-One-Winter19/nukes-and-origami#9]
Maybe some cool display of the text "nukes and origami" and have a flashing start button.

We also would get focus for controls for free!

**Comment by michaelosbornegit** [comment]
Also to clarify, I mean this will be the first screen the user sees when opening the game.

**Comment by jim-rosales** [comment]
Yup makes sense I was thinking that the plane would fly in and idle like the animation Nathan first created until the user pressed the start button
